src/html/discount.html

```diff
@@ -0,0 +1,116 @@
+<div class="discount">
+    <div class="container">
```

Comment:
<div class="discount">
    <div class="container">

can be replaced with just:

<div class="discount container">